deepfence_server/reporters/scan/scan_result_actions.go

```diff
@@ -239,54 +240,60 @@ func StopCloudComplianceScan(ctx context.Context, scanType, scanId string) error
 	defer tx.Close()
 
 	query := `MATCH (n:%s) -[:SCANNED]-> ()
-        WHERE n.node_id = $scan_id
+        WHERE n.node_id = $scanid
```

Comment:
When access by `node_id` we use this form:
```
MATCH (n:%s{node_id: $scanid})
```
It's clearer as it show it's a primary key, let's keep `where` clauses for all other fields.

Also please keep `scan_id` instead of `scanid`

Reply:
Done.